version: "3"

services:
  pyblock-node1-service:
    image: badlog/blockpy-node1
    container_name: blockchain-py-server-node1
    build:
      context: ./
    volumes:
      - ./src:/usr/app/src
    environment:
      - PORT=5001
    ports:
      - 5001:5001
    restart: on-failure
